buildchain/buildchain/salt_tree.py

```diff
@@ -227,6 +227,7 @@ def _get_parts(self) -> Iterator[str]:
     Path('salt/metalk8s/addons/prometheus-operator/deployed/init.sls'),
     Path('salt/metalk8s/addons/prometheus-operator/deployed/namespace.sls'),
     Path('salt/metalk8s/addons/prometheus-operator/deployed/storageclass.sls'),
+    Path('salt/metalk8s/addons/prometheus-operator/deployed/post-cleanup.sls'),
```

Comment:
Please move this file outside from `deployed` (it's the same as `pre-upgrade` `pre-downgrade`)